extensions/firefox/content/PdfStreamConverter.jsm

```diff
@@ -795,41 +810,28 @@ FindEventManager.prototype.bind = function() {
   }.bind(this);
   this.contentWindow.addEventListener('unload', unload);
 
-  for (var i = 0; i < this.types.length; i++) {
-    var type = this.types[i];
-    this.eventElement.addEventListener(type, this, true);
-  }
+  // We cannot directly attach listeners to for the find events
```

Comment:
That is weird. getChromeWindow and getFindBar from PdfjsContentUtils.jsm should never be called, in fact it's a bug that I have not removed them like I did for their counterpart from PdfjsChromeUtils.jsm ... Do you by any chance know who is the caller when that happens?